docs/src/main/docs/health/01_health.adoc

```diff
@@ -20,71 +20,191 @@
 :description: Helidon health checks
 :keywords: helidon, health-checks, health, check
 
-== Overview
-Helidon SE provides the following to support health checks:
+This document describes the health check API available with Helidon SE.
 
-1. The endpoint `/health`: A configurable endpoint that exposes health checks in JSON
-    format as described by the MicroProfile Health specification
-2. A base set of health checks (a separate library `io.helidon.health:helidon-health-checks`)
+== About health checks
 
-The Health Check component in Helidon SE is the core for the Helidon MP implementation of the MicroProfile Health specification.
+It's a good practice to monitor your microservice's health in order to ensure
+ it is available and performs correctly.
 
+Applications implement health checks to expose health status that is collected
+ at regular intervals by external tooling such as orchestrators like Kubernetes.
```

Comment:
Add a comma before "such as orchestrators like Kubernetes" just so readers get an opportunity to catch their breath in this rather long sentence :-)

Reply:
Fixed.